PartyWebApp/Controllers/PartiesController.cs

```diff
@@ -20,9 +20,16 @@ public PartiesController(PartyWebAppContext context)
         }
 
         // GET: Parties
-        public async Task<IActionResult> Index()
+        public async Task<IActionResult> Index(string searchString)
```

Comment:
I think it's a little bit useless to call the variable searchString because you define it as string in the first place..
Maybe calling it searchInput to emphasize that the user has given it?